docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,451 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# T<sub>2</sub> Hahn Characterization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The purpose of the $T_2$ Hahn Echo experiment is to determine $T_2$ qubit property. \n",
+    "\n",
+    "In this experiment, we would like to get a more precise estimate of the qubit's decay time. $T_2$ represents the amount of time required for the transverse magnetization to fall to approximately 37% ($\\frac{1}{e}$) of its initial value.\n",
+    "\n",
+    "Since the qubit is exposed to other types of noise (like $T_1$), we are using a $Rx(\\pi)$ pulse for decoupling and to solve our inaccuracy for the qubit frequency estimation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import qiskit\n",
+    "from qiskit_experiments.library.characterization.t2hahn import T2Hahn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The circuit used for the experiment comprises the following:\n",
+    "\n",
+    "    1. Rx gate\n",
+    "    2. delay\n",
+    "    3. measurement\n",
+    "\n",
+    "The user provides as input a series of delays in seconds. During the delay, we expect the qubit to precess about the z-axis. Because of the echo gate ($Rx(\\pi)$) for each echo, the angle after the delay gates will be $\\theta_{new} = \\theta_{old} + \\pi$. after wating the same delay time, the angle will be approximly $0$ or $\\pi$. By varying the extension of the delays, we get a series of decaying measurements. We can draw the graph of the resulting function and can analytically extract the desired values."
```

Comment:
```suggestion
    "The user provides as input a series of delays in seconds. During the delay, we expect the qubit to precess about the z-axis. Because of the echo gate ($Rx(\\pi)$) for each echo, the angle after the delay gates will be $\\theta_{new} = \\theta_{old} + \\pi$. After waiting the same delay time, the angle will be approximately $0$ or $\\pi$. By varying the extension of the delays, we get a series of decaying measurements. We can draw the graph of the resulting function and can analytically extract the desired values."
```

Reply:
Changed as suggested.